plugins/org.eclipse.xtext/src/org/eclipse/xtext/service/DefaultRuntimeModule.java

```diff
@@ -240,4 +242,12 @@ public void configureGenericSemanticSequencer(com.google.inject.Binder binder) {
 	public void configureUseIndexFragmentsForLazyLinking(com.google.inject.Binder binder) {
 		binder.bind(Boolean.TYPE).annotatedWith(Names.named(LazyURIEncoder.USE_INDEXED_FRAGMENTS_BINDING)).toInstance(Boolean.TRUE);
 	}
+	
+	
+	/**
+	 * @since 2.9
+	 */
+	public Class<? extends IWorkspaceConfigProvider> bindWorkspaceConfigProvider() {
+		return RuntimeWorkspaceConfigProvider.class;
```

Comment:
Should be symmetric to DefaultUI, we may not bind it there, so probably shouldn't bind it here neither.